webapp-django/crashstats/cron/management/commands/cronlist.py

```diff
@@ -15,6 +15,17 @@ class Command(BaseCommand):
     def handle(self, **options):
         for job_spec in JOBS:
             self.stdout.write(job_spec['cmd'])
+
+            cmdline = [job_spec['cmd']]
+            if job_spec.get('cmd_args'):
+                cmdline.extend(job_spec['cmd_args'])
+            if job_spec.get('backfill'):
+                cmdline.append('--run-time=RUNTIME')
+            if job_spec.get('last_success'):
+                cmdline.append('--last-success=LASTSUCCESS')
+            cmdline = ' '.join(cmdline)
+            self.stdout.write('    cmdline:      %s' % cmdline)
```

Comment:
This code is sort of the same as the code in `cronrun`, but it's for displaying the command line--not running. So the output is a little different. Otherwise I would have surfaced it because it sort of computes the same thing and now we have it in two places. We can revisit this later.

Seeing the full command line will be AWESOME for backfill jobs and jobs that do things depending on their last success.